staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -430,8 +420,6 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		actions = appendIf(actions, action{"LIST", resourcePath, resourceParams, namer, false}, isLister)
 		actions = appendIf(actions, action{"POST", resourcePath, resourceParams, namer, false}, isCreater)
 		actions = appendIf(actions, action{"DELETECOLLECTION", resourcePath, resourceParams, namer, false}, isCollectionDeleter)
-		// DEPRECATED in 1.11
-		actions = appendIf(actions, action{"WATCHLIST", "watch/" + resourcePath, resourceParams, namer, false}, allowWatchList)
```

Comment:
I would like to see this change, but I'm not sure if it's compatible with our policies...

@liggitt will be able to confirm

Reply:
it's in bounds per deprecation policies

it looks like some kubernetes org clients still use the deprecated watch paths - https://grep.app/search?q=watch/&filter[repo.pattern][0]=kubernetes-client/